tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "forceConsistentCasingInFileNames": true,
+    "isolatedModules": true,
+    "jsx": "react",
+    "lib": ["es6", "dom"],
+    "module": "commonjs",
+    "noEmit": true,
+    "strict": true,
```

Just a suggestion: I get a lot of errors around conflicting types, generally around react-native and the `dom` types. 

```suggestion
    "skipDefaultLibCheck": true,
    "strict": true,
```